examples/amp-story/bookendv1.json

```diff
@@ -61,6 +61,23 @@
       "title": "This is an example article",
       "url": "http://example.com/article.html",
       "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "cta-link",
+      "links": [
+        {
```

Comment:
Feature request:
If there is only one link, it would take the full width. I could see that being a common use case.
Eg: newsletter subscribe button

(Not in this PR)